lms/templates/fields/field_readonly_account.underscore

```diff
@@ -0,0 +1,11 @@
+<div class="field">
+    <% if (title) { %>
+        <span class="u-field-title field-label"><%- title %></span>
```

Comment:
add `aria-hidden="true"` as it will read at https://github.com/edx/edx-platform/pull/12369/files#diff-7e8b3256141d781f52e13710109200a5R5